### DISCUSSION TITLE
default album art?

### DISCUSSION THREAD

**Discussion by ghb1912** [jcorporation/myMPD#979] [Q&A] [answered]
Hello -

In previous versions of MyMPD (9.x), it was possible to supply and specify an image file which would be displayed during playback in cases where album art is not otherwise available from the sound file or on the filesystem. In 10.x, this no longer works, and I can't find any mention of it in the documentation. 

Is it still possible to do this? I miss being able to look at something other than the broken-image placeholder for the majority of my music library!

Thank you!

**Comment by jcorporation** [comment, marked as answer]
This feature is indeed not available in newer myMPD versions, but it could be readded.

**Reply by ghb1912** [reply]
If you are so inclined, it would be a very welcome feature. 

Thank you!

**Comment by Emilot** [comment]
+1 to readded...

**Comment by jcorporation** [comment]
Feature is already implemented in the v10.3.0 feature branch: https://github.com/jcorporation/myMPD/issues/980

**Reply by ghb1912** [reply]
That is pretty cool. Thanks again for a great piece of software!
